integrations/whatsapp/integration.definition.ts

```diff
@@ -63,7 +71,29 @@ export default new IntegrationDefinition({
       name: {},
     },
   },
-  actions: {},
+  actions: {
+    startConversation: {
+      title: 'Start Conversation',
+      description:
+        "Starts a conversation with a user's Whatsapp phone number by sending them a message using a Whatsapp Message Template.",
+      input: {
+        schema: z.object({
+          userPhone: z.string().describe(TagsForCreatingConversation.userPhone.description),
+          templateName: z.string().describe(TagsForCreatingConversation.templateName.description),
+          templateLanguage: z.string().optional().describe(TagsForCreatingConversation.templateLanguage.description),
+          templateVariablesJson: z
```

Comment:
**chore:** Why do we pass a json string here ? Why not a record ? It's kindof weird

Reply:
I think the Studio only supports plain text values for action inputs?